src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -203,20 +212,22 @@ public void toModelType_invalidAppointments_throwsIllegalValueException() {
         invalidAppointments.add(new JsonAdaptedAppointment(INVALID_APPOINTMENTS));
         JsonAdaptedPerson person =
                 new JsonAdaptedPerson(VALID_NAME, VALID_PHONE, VALID_EMAIL, VALID_ADDRESS, VALID_ADDRESS,
-                    VALID_DATE_OF_BIRTH, VALID_GENDER, VALID_TAGS, VALID_PRIORITY, invalidAppointments);
+                    VALID_DATE_OF_BIRTH, VALID_GENDER, VALID_TAGS, VALID_PRIORITY, invalidAppointments, VALID_MEDCON);
     }
     @Test
     public void toModelType_invalidPriority_throwsIllegalValueException() {
         JsonAdaptedPerson person = new JsonAdaptedPerson(VALID_NAME, VALID_PHONE, VALID_EMAIL, VALID_ADDRESS,
-                VALID_DATE_OF_BIRTH, VALID_GENDER, VALID_NRIC, VALID_TAGS, INVALID_PRIORITY, VALID_APPOINTMENTS);
+                VALID_DATE_OF_BIRTH, VALID_GENDER, VALID_NRIC, VALID_TAGS, INVALID_PRIORITY, VALID_APPOINTMENTS,
+                VALID_MEDCON);
         String expectedMessage = Priority.MESSAGE_CONSTRAINTS;
         assertThrows(IllegalArgumentException.class, expectedMessage, person::toModelType);
     }
 
     @Test
     public void toModelType_nullPriority_throwsIllegalValueException() {
         JsonAdaptedPerson person = new JsonAdaptedPerson(VALID_NAME, VALID_PHONE, VALID_EMAIL, VALID_ADDRESS,
-                VALID_DATE_OF_BIRTH, VALID_GENDER, VALID_NRIC, VALID_TAGS, null, VALID_APPOINTMENTS);
+                VALID_DATE_OF_BIRTH, VALID_GENDER, VALID_NRIC, VALID_TAGS, null, VALID_APPOINTMENTS,
+                VALID_MEDCON);
         String expectedMessage = String.format(MISSING_FIELD_MESSAGE_FORMAT, Priority.class.getSimpleName());
         assertThrows(IllegalValueException.class, expectedMessage, person::toModelType);
     }
```

Comment:
Consider adding test case for invalid medCon and null medCone

Reply:
Thing is there isn't really any. Rmb we agreed even skibidi is a valid medCon haha. But I will update this later such that users can input multiple medCon soon